.github/workflows/laravel.yml

```diff
@@ -0,0 +1,31 @@
+name: Laravel
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master, next ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Copy .env
+      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
+    - name: Generate key
+      run: php artisan key:generate --force
+    - name: Directory Permissions
+      run: chmod -R 777 storage bootstrap/cache
+    - name: Create Database
+      run: |
+        mkdir -p database
+        touch database/twilio_verify_quickstart.sqlite
```

Comment:
PHPUnit is setup to use in-memory SQLite DB. This step shouldn't be required.
https://github.com/TwilioDevEd/verify-v2-quickstart-php/blob/master/phpunit.xml#L32